nx-libs.spec

```diff
@@ -19,6 +19,7 @@ BuildRequires:  gcc-c++
 BuildRequires:  libjpeg-devel
 BuildRequires:  pkgconfig
 BuildRequires:  imake
+BuildRequires:  quilt
```

Comment:
Hi,

quilt is nice for creating and managing patches. but in the end there are patches. So why not apply the patches on rh and suse with the patch command directly. This will cause only some overhead.